packages/trackerless-network/README.md

```diff
@@ -1,6 +1,63 @@
 # Trackerless Network
 
+A P2P topic publish-subscribe network.
+
 The Trackerless Network package is a reimplementation of the old (Corea-Brubeck) network package. 
 The package is reimplemented to use the network and transport stacks of the proto-rpc and DHT packages.
 The main change to the network is that the `d-regular random graph` stream topologies are now generated 
 using a decentralized algorithm based on peer discovery from the DHT.
+
+## Running a node
+
+Running a network node requires a DhtNode from the `@streamr/dht` to use as control layer. For more details on how to configure the DHT node go [here](packages/dht/README.md) The control layer node is configured as follows:
+
+```js
+const networkNode = new NetworkNode({
+    // Give all control layer DHT configs here.
+    layer0: {
+
+    },
+    // Content layer specific configurations:
```

Comment:
Content layer is a good name 👍 Does it refer to layer1 + layer2, i.e. the layers which we've been calling stream control layer + stream delivery layer?

Reply:
My thought here is that it would refer to both.